drivers/sensors/Kconfig

```diff
@@ -1145,4 +1145,21 @@ config HDC1008_DEBUG
 		Enables debug features for the HDC1008
 
 endif # SENSORS_HDC1008
+
+config SENSORS_LTR308
+	bool "LTR308 Lite-On ambient light sensor"
+	default n
+  select I2C
+	---help---
+		Enable driver support for the LTR308 Lite-on ambient light sensor.
+
+if SENSORS_LTR308
+
+config LTR308_I2C_FREQUENCY
```

Comment:
Do we need to add configuration for i2c address? Or does the sensor have a fixed address?

Reply:
@robertalexa2000 Petro is right, this sensor has fixed address (0x53) see: https://optoelectronics.liteon.com/upload/download/DS86-2016-0027/LTR-308ALS_Final_%20DS_V1%201.pdf

Reply:
@pkarashchenko I've tagged you in another comment regarding the i2c address